lib/whitehall_importer/integrity_checker.rb

```diff
@@ -67,6 +67,14 @@ def time_problems
                                         first_published_at,
                                         proposed_first_published_at)
       end
+      proposed_public_updated_at = proposed_payload["public_updated_at"]
+      public_updated_at = publishing_api_content["public_updated_at"]
+
+      unless edition.draft? && edition.first? || time_similar?(proposed_public_updated_at, public_updated_at)
```

Comment:
This is a pretty tough unless to understand - I personally stay clear of using unless with multiple things because the logic can be tough to parse. 

I'm also wondering if we have a logic issue here as I think it's not that something is draft that we care about but instead that it's not live (as submitted for review and scheduled would also have this issue).

Maybe try: `if edition.live? && time_similar?` ? This wouldn't check non first drafts but that's probably fine anyway as the live versions will be checked.

Reply:
Good spot Kev, I've updated the logic to `if edition.live? && !time_similar?`  which is what I think you meant? 

This wouldn't check editions that are not live and would only raise a problem if the edition is live and the times are not similar.